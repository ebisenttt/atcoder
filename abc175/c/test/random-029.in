-153961756406217 80574679610766 288617990390119
