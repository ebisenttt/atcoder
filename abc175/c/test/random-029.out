153961756406217
